### PR TITLE
[avanyan-staked-in-vaults] anyan-staked-in-vaults added

### PR DESCRIPTION
**Pull request by avanyanio**
Changes proposed in this pull request:
- Previous pull request closed at https://github.com/snapshot-labs/snapshot-strategies/pull/88 and resend one by one. 
- Also added strategy to this filehttps://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/index.ts
-

### CODE REVIEW

File: src/strategies/avanyan-staked-in-vaults/index.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'avanyan';
```
Comment (ChaituVR) [inline src/strategies/avanyan-staked-in-vaults/index.ts:4]:
```suggestion
export const author = 'avanyanio';
```

File: src/strategies/index.ts [hunk 2]
```diff
@@ -154,6 +154,7 @@ import * as occStakeOf from './occ-stake-of';
 import * as hoprStaking from './hopr-staking';
 import * as hoprBridgedBalance from './hopr-bridged-balance';
 import * as lootCharacterGuilds from './loot-character-guilds';
+import * as anyanVaults from './anyan-staked-in-vaults';
```
Comment (ChaituVR) [inline src/strategies/index.ts:157]: The folder name is different to strategy

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/index.ts (hunk 2 above)
Comment (avanyanio) [reply src/strategies/index.ts]: Sorry, I completely lost my concentration when I was working on it, corrected.

**Comment by avanyanio** [comment]
- Author editted as per suggested.
- Folder is corrected

**Comment by avanyanio** [comment]
@ChaituVR Sir, I am stucked about test strategies. Can you guide me please? I am kinda newbie programmer and confused.

**Comment by ChaituVR** [comment]
@avanyanio best way to test strategies is cloning this fork repo locally, 
Try running:
`npm test --strategy=avanyan-staked-in-vaults`

**Comment by avanyanio** [comment]
Will restart coding.

**Pull request closed without merging**